custom_components/tuya_local/devices/smartplug_usb_childlock.yaml

```diff
@@ -0,0 +1,95 @@
+name: Plug In WiFi Smart Socket with USB
+products:
+  - id: ynz9qmnmnsciccyc
+    name: Plug In WiFi Smart Socket with USB
+primary_entity:
+  entity: switch
+  class: outlet
+  name: USB
+  icon: "mdi:usb"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: switch
+    name: AC Outlet
```

Comment:
I would leave this out and let the class translations define the name, since this is a normal outlet and there is only one.